Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libnfs"
-version = "0.1.1"
+version = "2.0.0"
```

Comment:
If we bumped this to 0.2 I think it would also allow breaking changes. After it hits 1.0 we're saying it's stable and then it would need a major bump again for breaking changes.

Reply:
@cholcombe973

Sounds reasonable. I can probably update the workflow to clone and build the library, which is required for this, I will take a look later.

I might update this PR a few times in an attempt to get the pipeline working, ignore failures and I will ping you when I think it merits attention.